t/mojo/morbo.t

```diff
@@ -153,6 +154,53 @@ subtest 'Broken symlink' => sub {
   ok !$warned, 'no warnings';
 };
 
+subtest 'Randomized Request IDs' => sub {
+  my @ids;
+  $script->spurt(<<"EOF");
+use Mojolicious::Lite;
+use Mojo::File qw(path);
+use Mojo::IOLoop;
+
+app->log->level('fatal');
+
+Mojo::IOLoop->next_tick(sub { path('$started')->touch });
+
+# Return "request_id"
+get '/request_id' => sub {
+  my \$c = shift;
+  \$c->render( text => \$c->req->request_id );
+};
+
+app->start;
+EOF
+  sleep 0.1 until -f $started;
+
+  # Fetch initial request_ids
+  my $tx = $ua->get("http://127.0.0.1:$port/request_id");
+  like $tx->res->body, qr/^[\-_0-9a-z]+$/i, 'request_id is valid';
```

Comment:
This does not seem the right place to test the actual format of the request id, which we change every now and then. Only makes that harder in the future.